getting-started/5_start_ui.md

```diff
@@ -2,15 +2,19 @@ Back in our original terminal, we can now start the UI. Before we do so, we need
 
 First, add the below line to `ui/.env`{{open}}
 
-<pre class="file" data-target="clipboard">DANGEROUSLY_DISABLE_HOST_CHECK=true
+<pre class="file" data-filename="ui/.env" data-target="append">DANGEROUSLY_DISABLE_HOST_CHECK=true
 </pre>
 
 Secondly, edit the websocket URL in `ui/src/config.ts`{{open}} to 
 
 <pre class="file" data-target="clipboard">wss://[[HOST_SUBDOMAIN]]-7575-[[KATACODA_HOST]].environments.katacoda.com/</pre>
 
+To do so, you can run
+
 ```
-yarn start
+sed -i 's+ws://localhost:7575/+wss://[[HOST_SUBDOMAIN]]-7575-[[KATACODA_HOST]].environments.katacoda.com/+g' src/config.ts
```

Comment:
The Editor integration doesn't support RegExes yet. Once it does, I will replace this with a nice button.